scripts/releaseTag.mjs

```diff
@@ -40,12 +42,12 @@ async function findMuiOrgRemote() {
       // - git@github.com:mui/material-ui.git
       // but not:
       // - git@github.com:mui/material-ui-docs.git
-      return /mui\/material-ui(\.git)?$/.test(remote.url) && remote.method === '(push)';
+      return new RegExp(`mui/${repo}(.git)?$`).test(remote.url) && remote.method === '(push)';
```

Comment:
Wouldn't be surprised there's some hidden bugs in this regex still. My advice would be to always avoid regex if proper parsers exist. What do you think about using [`parse-github-url`](https://www.npmjs.com/package/parse-github-url) instead?

Reply:
```
mui/${repo}(.git)?$
------------^ this is an "anything" wildcard
            | mui/x-chartssgit would match
```

Reply:
Updated to use the parser.

Reply:
It would also match "not-mui/material-ui.git". 🙂